packages/back-end/src/services/experiments.ts

```diff
@@ -1031,6 +1031,28 @@ export function postMetricApiPayloadIsValid(
             "`behavior.maxPercentChange` must be greater than `behavior.minPercentChange`",
         };
     }
+
+    // Check capping args + capping values
+    const { capping, capValue } = behavior;
+
+    const cappingExists = typeof capping !== "undefined";
+    const capValueExists = typeof capValue !== "undefined";
+    if (cappingExists !== capValueExists) {
+      return {
+        valid: false,
+        error:
+          "Must specify both `behavior.capping` and `behavior.capValue or neither.",
+      };
+    }
+    if (capping && capValue && capping === "percentile") {
+      if (capValue > 1) {
```

Comment:
these 2 lines can be:
`if (capping === "percentile" && capValue > 1)`
or if capValue could be undefined, then:
`if (capping === "percentile" && (capValue || 0) > 1`